### PR TITLE
feat(api-reference): re-support `x-scalar-sdk-installation` source

### PR DESCRIPTION
**Pull request by hanspagel**
We dropped `source` from `x-scalar-sdk-installation` when the block moved to a single Markdown `description`. This brings it back without un-promoting `description`.

When an entry has a `source`, it is folded into the rendered `description` as a fenced code block. If there is no `description`, the `source` becomes the description on its own. The fence widens past any backticks inside `source`, so a command containing backticks still renders as one code block.

`source` is marked `@deprecated` in the schema and generated types, so docs and editors keep steering people toward `description`.

---

> [!NOTE]
> **Low Risk**
> Documentation rendering and schema backward compatibility only; behavior is covered by unit tests and does not touch auth or data paths.
> 
> **Overview**
> Brings back the deprecated **`source`** install field on **`x-scalar-sdk-installation`** while keeping **`description`** as the primary Markdown field. Schemas, workspace-store validation, and generated types now accept optional **`source`** again and mark it **`@deprecated`**.
> 
> **`getRenderableSdks`** now resolves each entry to a single renderable **`description`**: legacy **`source`** alone becomes a fenced code block, or it is appended after **`description`** when both are set. Fence length grows past backticks inside **`source`** so nested fences still render safely; whitespace-only **`description`** is ignored but non-empty descriptions stay verbatim. Entries with neither field are still dropped.
> 
> Tests cover parsing, merging, backtick edge cases, and empty entries; a changeset bumps the affected packages.
> 
> <sup>Reviewed by [Cursor Bugbot](https://cursor.com/bugbot) for commit 463c8e2c207a8fbd7e6f118abd6fc1646c4bb10c. Bugbot is set up for automated code reviews on this repo. Configure [here](https://www.cursor.com/dashboard/bugbot).</sup>

### CODE REVIEW

**Review by marclave — APPROVED** [review]

File: packages/api-reference/src/blocks/scalar-sdk-installation-instructions/helpers/renderable-sdks.ts [hunk 1]
```diff
@@ -3,22 +3,59 @@ import type { XScalarSdkInstallation } from '@scalar/workspace-store/schemas/ext
...
+        }
+
+        const description = typeof sdk.description === 'string' ? sdk.description.trim() : ''
+        const source = typeof sdk.source === 'string' ? sdk.source.trim() : ''
+
+        if (!description && !source) {
+          return []
+        }
+
+        const resolved = [description, source && toFencedCodeBlock(source)].filter(Boolean).join('\n\n')
```
[the remark this answers is not in the document]
Comment (hanspagel) [reply packages/api-reference/src/blocks/scalar-sdk-installation-instructions/helpers/renderable-sdks.ts:60]: Good catch — fixed in 463c8e2c2. The description is now rendered verbatim; the trimmed copy is only used for the empty-check, so indented code blocks and other significant whitespace are preserved.
